concourse/pipeline.yml

```diff
@@ -141,7 +141,7 @@ jobs:
       - task: run-smoke-tests
         file: govuk-coronavirus-business-volunteer-form/concourse/tasks/run-smoke-tests.yml
         params:
-          TEST_URL: 'https://coronavirus-business-volunteers.service.gov.uk/medical-equipment'
+          TEST_URL: 'https://coronavirus-business-volunteers.service.gov.uk'
```

Comment:
This should go back to the way it was. The `/` route redirects back to GOV.UK, and [we don't have any feature tests that click "Start now"](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/f334af7ffe7ebc29faffe444ef1dab4d38588f53/spec/support/fill_in_the_form_steps.rb#L4-L6). Probably want to remove the first feature, actually...

Reply:
Ah sorry I should've separated this out into another commit for clarity. This change is intended, since `TEST_URL` is used by the feature tests as the `Capybara.app_host`, so when one writes `visit "medical_equipment_path"` behind the scenes this gets translated into `Capybara.app_host + path` or `https://coronavirus-business-volunteers.service.gov.uk/medical-equipment`.

The previous commit was broken, so if I left it as-is the calls to `visit_path` would take the test browser to e.g. `/medical-equipment/medical-equipment/`.

Reply:
Ah, that makes more sense!